test/TestConfig/src/org/openj9/envInfo/MachineInfo.java

```diff
@@ -0,0 +1,283 @@
+		return "\nuname: " + uname
+		+ "\ncpuCores: " + cpuCores 
+		+ "\nsysArch: " + sysArch 
+		+ "\nprocArch: " + procArch 
+		+ "\nsysOS: " + sysOS
+		+ "\n"
+		+ "\nvmVendor: " + vmVendor 
+		+ "\nvmVersion: " + vmVersion 
+		+ "\nspecVendor: " + specVendor 
+		+ "\nspecVersion: " + specVersion 
+		+ "\njavaVersion: " + javaVersion 
+		+ "\n"
+		+"\nTotal memory (bytes): " + totalMemory
+		+"\nFree memory (bytes): " + freeMemory
+		+ "\n";
+	}
+
+	public long getFreeMemory() {
+		return freeMemory;
+	}
+
+	public void setFreeMemory(long freeMemory) {
+		this.freeMemory = freeMemory;
+	}
+
+	public long getTotalMemory() {
+		return totalMemory;
+	}
+
+	public void setTotalMemory(long totalMemory) {
+		this.totalMemory = totalMemory;
+	}
+
+	public String getUname() {
+		return uname;
+	}
+
+	public void setUname(String uname) {
+		this.uname = uname;
+	}
+
+	public String getVmVendor() {
+		return vmVendor;
+	}
+
+	public void setVmVendor(String vmVendor) {
+		this.vmVendor = vmVendor;
+	}
+
+	public String getVmVersion() {
+		return vmVersion;
+	}
+
+	public void setVmVersion(String vmVersion) {
+		this.vmVersion = vmVersion;
+	}
+
+	public String getSpecVendor() {
+		return specVendor;
+	}
+
+	public void setSpecVendor(String specVendor) {
+		this.specVendor = specVendor;
+	}
+
+	public String getSpecVersion() {
+		return specVersion;
+	}
+
+	public void setSpecVersion(String specVersion) {
+		this.specVersion = specVersion;
+	}
+
+	public String getJavaVersion() {
+		return javaVersion;
+	}
+
+	public void setJavaVersion(String javaVersion) {
+		this.javaVersion = javaVersion;
+	}
+
+	public String getInstalledMem() {
+		return installedMem;
+	}
+
+	public void setInstalledMem(String installedMem) {
+		this.installedMem = installedMem;
+	}
+
+	public String getFreeMem() {
+		return freeMem;
+	}
+
+	public void setFreeMem(String freeMem) {
+		this.freeMem = freeMem;
+	}
+
+	public String getCpuCores() {
+		return cpuCores;
+	}
+
+	public void setCpuCores(String cpuCores) {
+		this.cpuCores = cpuCores;
+	}
+
+	public String getCpuModel() {
+		return cpuModel;
+	}
+
+	public void setCpuModel(String cpuModel) {
+		this.cpuModel = cpuModel;
+	}
+
+	public String getCpuSpeed() {
+		return cpuSpeed;
+	}
+
+	public void setCpuSpeed(String cpuSpeed) {
+		this.cpuSpeed = cpuSpeed;
+	}
+
+	public String getSysArch() {
+		return sysArch;
+	}
+
+	public void setSysArch(String sysArch) {
+		this.sysArch = sysArch;
+	}
+
+	public String getSysOS() {
+		return sysOS;
+	}
+
+	public void setSysOS(String sysOS) {
+		this.sysOS = sysOS;
+	}
+
+	public String getProcArch() {
+		return procArch;
+	}
+
+	public void setProcArch(String procArch) {
+		this.procArch = procArch;
+	}
+
+	public String getNuma() {
+		return numa;
+	}
+
+	public void setNuma(String numa) {
+		this.numa = numa;
+	}
+
+	public String getSysVirt() {
+		return sysVirt;
+	}
+
+	public void setSysVirt(String sysVirt) {
+		this.sysVirt = sysVirt;
+	}
+
+	public void getRuntimeInfo() {
```

Comment:
Probably something a bit off about the naming of the methods (my bad, handed you some pretty rough code), since this getRuntimeInfo, is actually calling a bunch of setter methods... storeRuntimeInfo?  Feel free to rename or adjust as you see fit.

Same story for getMachineInfo.